Containers/nextcloud/entrypoint.sh

```diff
@@ -537,8 +537,10 @@ fi
 chmod 775 -R /var/www/html/custom_apps/notify_push/bin/
 php /var/www/html/occ config:system:set trusted_proxies 0 --value="127.0.0.1"
 php /var/www/html/occ config:system:set trusted_proxies 1 --value="::1"
+# IP of docker network gateway
+php /var/www/html/occ config:system:set trusted_proxies 2 --value="$(ip r | grep default | grep -o "[0-9]\+\.[0-9]\+\.[0-9]\+\.1")"
```

Comment:
I fear this will overwrite the changes some people might have done already: https://github.com/nextcloud/all-in-one/discussions/2045
So we should probably choose another array key?

Reply:
Also should we maybe use the same logic like here? https://github.com/nextcloud/all-in-one/blob/17b35a45db08951e7f86e09a971563b407842cc9/Containers/apache/start.sh#L20-L24

Reply:
> Also should we maybe use the same logic like here?
> 
> https://github.com/nextcloud/all-in-one/blob/17b35a45db08951e7f86e09a971563b407842cc9/Containers/apache/start.sh#L20-L24

Seems to also work without